.env

```diff
@@ -34,7 +34,7 @@ REACT_APP_KLEROS_LIQUID_SOKOL_ADDRESS=0xb701ff19fBD9702DD7Ca099Ee7D0D42a2612baB5
 REACT_APP_KLEROS_LIQUID_EXTRA_VIEWS_SOKOL_ADDRESS=0x1Aab57DD59afc98bff1D939b34974E9c8fc705fF
 REACT_APP_PINAKION_SOKOL_ADDRESS=0x9Bc02E9f5291adbb1f0Cf5C15ab1B400dccd3665
 REACT_APP_RAW_PINAKION_SOKOL_ADDRESS=0x184A7Fc4fa965D18Af84C6d97dfed8C4561ff8c2
-REACT_APP_POLICY_REGISTRY_SOKOL_ADDRESS=0x0Bee63bC7220d0Bacd8A3c9d6B6511126CDfe58f
+REACT_APP_POLICY_REGISTRY_SOKOL_ADDRESS=0x933FFDaa817A1F981A1FA089240b53a7a79f482a
```

Comment:
@hbarcelos this is the policy registry address, not the kleros liquid extra views address.

Reply:
Hey man, thanks for the heads up.
I'm gonna close this PR because this ended up in another branch.